### ISSUE TITLE
SEACAS Tests Failing on Clean Track

### ISSUE THREAD

**Issue by william76** [trilinos/Trilinos#2042]
There are 3 SEACAS tests failing on the nightlies (https://testing.sandia.gov/cdash/viewTest.php?onlyfailed&buildid=3253872)
- [SEACASAprepro_aprepro_unit_test](https://testing.sandia.gov/cdash/testDetails.php?test=43150010&build=3254222)
- [SEACASExodus_exodus_unit_tests](https://testing.sandia.gov/cdash/testDetails.php?test=43188945&build=3254222)
- [SEACASExodus_exodus_unit_tests_nc5_env](https://testing.sandia.gov/cdash/testDetails.php?test=43188946&build=3254222)

@trilinos/seacas 

## Expectations
Test failures on the CLEAN track block promotion from develop to master and thus should be fixed promptly.

## Related Issues
Possibly related to the commits that broke the CI builds on issue #2039 ?

## Additional Information
Console outputs on CDash: 
- https://testing.sandia.gov/cdash/testDetails.php?test=43150010&build=3254222
- https://testing.sandia.gov/cdash/testDetails.php?test=43188945&build=3254222
- https://testing.sandia.gov/cdash/testDetails.php?test=43188946&build=3254222

**Comment by gsjaardema** [comment]
The problem is that the failing tests are not correctly implemented to permit parallel test runs.  I have fixed them in SEACAS github and am pushing to Sierra so they can be synced to Trilinos, but that may take about a week.

**Comment by bmpersc** [comment]
I have reverted the STK and SEACAS integration commits which should fix this. I'll let the nightly tests run to make sure that things are back to the previous behavior before closing this.

**Comment by gsjaardema** [comment]
I don't think that the commit had anything to do with causing the tests to fail in parallel test runs.  The tests should have been failing in parallel even prior to that commit, so reverting it probably won't fix anything.
